src/test/script_tests.cpp

```diff
@@ -2445,40 +2445,7 @@ BOOST_AUTO_TEST_CASE(script_json_test) {
 }
 
 BOOST_AUTO_TEST_CASE(script_PushData) {
-    // Check that PUSHDATA1, PUSHDATA2, and PUSHDATA4 create the same value on
-    // the stack as the 1-75 opcodes do.
-    static const uint8_t direct[] = {1, 0x5a};
-    static const uint8_t pushdata1[] = {OP_PUSHDATA1, 1, 0x5a};
-    static const uint8_t pushdata2[] = {OP_PUSHDATA2, 1, 0, 0x5a};
-    static const uint8_t pushdata4[] = {OP_PUSHDATA4, 1, 0, 0, 0, 0x5a};
-
     ScriptError err;
-    std::vector<std::vector<uint8_t>> directStack;
-    BOOST_CHECK(EvalScript(directStack,
-                           CScript(direct, direct + sizeof(direct)),
-                           SCRIPT_VERIFY_P2SH, BaseSignatureChecker(), &err));
-    BOOST_CHECK_MESSAGE(err == ScriptError::OK, ScriptErrorString(err));
-
-    std::vector<std::vector<uint8_t>> pushdata1Stack;
-    BOOST_CHECK(EvalScript(pushdata1Stack,
-                           CScript(pushdata1, pushdata1 + sizeof(pushdata1)),
-                           SCRIPT_VERIFY_P2SH, BaseSignatureChecker(), &err));
-    BOOST_CHECK(pushdata1Stack == directStack);
-    BOOST_CHECK_MESSAGE(err == ScriptError::OK, ScriptErrorString(err));
-
-    std::vector<std::vector<uint8_t>> pushdata2Stack;
-    BOOST_CHECK(EvalScript(pushdata2Stack,
-                           CScript(pushdata2, pushdata2 + sizeof(pushdata2)),
-                           SCRIPT_VERIFY_P2SH, BaseSignatureChecker(), &err));
-    BOOST_CHECK(pushdata2Stack == directStack);
-    BOOST_CHECK_MESSAGE(err == ScriptError::OK, ScriptErrorString(err));
-
-    std::vector<std::vector<uint8_t>> pushdata4Stack;
-    BOOST_CHECK(EvalScript(pushdata4Stack,
-                           CScript(pushdata4, pushdata4 + sizeof(pushdata4)),
-                           SCRIPT_VERIFY_P2SH, BaseSignatureChecker(), &err));
-    BOOST_CHECK(pushdata4Stack == directStack);
-    BOOST_CHECK_MESSAGE(err == ScriptError::OK, ScriptErrorString(err));
```

Comment:
For context; these tests make no sense anymore, as there's only one single way to push data.